lib/elastic_apm/context/response.rb

```diff
@@ -38,9 +38,7 @@ def initialize(
       attr_reader :headers
 
       def headers=(headers)
-        @headers = headers&.each_with_object({}) do |(k, v), hsh|
-          hsh[k] = v.to_s
-        end
+        @headers = headers&.transform_values(&:to_s)
```

Comment:
I think `rubocop-performance` suggests using real blocks instead of symbol-to-proc?

Reply:
The only thing I could find was [this](https://docs.rubocop.org/rubocop-performance/cops_performance.html#performancemethodobjectasblock), which I don't believe is the same thing. RuboCop doesn't complain about this line.